local-dev/README.md

```diff
@@ -1,16 +1,28 @@
-# Setting up Audit Database
-
-When running locally, the audit endpoints require a Dynamo DB service to be running on port 8000.
-
-The service hosts a database holding a table that stores audit logs.
-
-To make this table available for testing auditing functionality:
+# Setting up a local Dynamo DB service
 
+You will need to have Docker installed to run the local Dynamo DB service.  
+Then:  
 1. Start Docker
 2. Start up a local Dynamo DB container:
 ```shell
 cd local-dev
 docker-compose up -d
 ```
-3. Create the audit table schema by running [Creation test](https://github.com/guardian/janus-app/blob/f522bca83d9d90657634b038c7734a9871974161/test/aws/AuditTrailDBTest.scala#L48-L50).
-4. Populate the table with some test data by running [Insertion test](https://github.com/guardian/janus-app/blob/f522bca83d9d90657634b038c7734a9871974161/test/aws/AuditTrailDBTest.scala#L16-L46).
+
+## Setting up Audit table
+
+The service hosts a database holding a table that stores audit logs.  
+To make this table available for testing auditing functionality:  
+1. Create the audit table schema by running [Creation test](https://github.com/guardian/janus-app/blob/f522bca83d9d90657634b038c7734a9871974161/test/aws/AuditTrailDBTest.scala#L48-L50).
+2. Populate the table with some test data by running [Insertion test](https://github.com/guardian/janus-app/blob/f522bca83d9d90657634b038c7734a9871974161/test/aws/AuditTrailDBTest.scala#L16-L46).
+
+
+## Setting up Passkeys tables
+
+To test registration of passkeys and their use in authentication, you will need to have the tables to hold passkey
+public key data and challenges in your local database.  
+To make these tables available for testing passkeys functionality:    
+1. Create the passkey credentials table schema by running [create table test](/test/aws/PasskeyDBTest.scala).
```

Comment:
```suggestion
1. Create the passkey credentials table schema by running [create table test](../test/aws/PasskeyDBTest.scala).
```

Reply:
The links do seem to work from the preview.  Doesn't the leading slash start the path from the repo root?

Reply:
Ah - they didn't work for me locally but you're right they work in the preview here!